apps/web/app/(auth-pages)/sign-up/page.tsx

```
@@ -116,37 +116,37 @@ export default function Signup() {
 							) : (
 								<div className="flex flex-col items-center justify-center">
-									<span>
+									<span role="alert" aria-live="assertive">
 										WebAuthn is not supported. Please use a different browser.
 									</span>
 								</div>
 							)}
 
 							{regSuccess && (
-								<div className="text-green-600">
+								<div className="text-green-600" role="status" aria-live="polite">
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)